data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -1010,6 +1010,13 @@ spec:
                           type: object
                         bootMACAddress:
                           type: string
+                        bootMode:
+                          description: BootMode is the way of launching the OS on
+                            the system
```

Comment:
Nit, I like the description in the install_ipi.md below better than this one

Reply:
Done